### ISSUE TITLE
Problems in evaluating under rgb-only categories on NTU120-CS.

### ISSUE THREAD

**Issue by Sfadingz** [dominickrei/pi-vit#9]
Hello, as you mentioned "(in your csv, just set the skeleton feature path to an empty string, they wont be used at inference)." in the other issue, do you mean setting the path_to_video_1_skeleton in the csv to an empty string? I did so, and got errors in getitem method of NTU dataset. When I keep the path_to_video_1_skeleton, I get a low top-1 accuracy of 66.23% on NTU120-CS, with the provided pretrained weights named NTU120_CS_pivit.pyth.

**Comment by dominickrei** [comment]
Hi @Sfadingz, I suspect this is because the full resolution version of NTU is being used for inference (I get a similar top-1 accuracy if I test on the full resolution videos). Because our models are trained on human cropped videos the input at inference is also expected to be performed on human cropped videos, which can be obtained using human detectors (e.g., YOLO).

Here is a script to perform the cropping using YOLO: https://gist.github.com/dominickrei/b88e035a426c27424f3b8e0c40a35f52

Thank you for your interest and I hope this helps!

**Comment by Sfadingz** [comment]
I processed the videos according to your script and got the accuracy in the paper. Thank you so much for your help!
